src/descriptor/policy.rs

```diff
@@ -76,6 +76,19 @@ pub enum PkOrF {
     XOnlyPubkey(XOnlyPublicKey),
     /// An extended key fingerprint
     Fingerprint(Fingerprint),
```

Comment:
Seems like we also don't need this `FingerPrint` anymore as its included in the XpubOrigin??

Reply:
Yep! Good catch.